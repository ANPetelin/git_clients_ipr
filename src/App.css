.App {
  ::-webkit-scrollbar {
    width: 6px;
    height: 6px;
  }

  /* Track */

  ::-webkit-scrollbar-track {
    background: transparent;
  }

  /* Handle */

  ::-webkit-scrollbar-thumb {
    background: #c1cbd8;
    border-radius: 2px;
  }

  /* Handle on hover */

  ::-webkit-scrollbar-thumb:hover {
    background: #929fb5;
  }  
}
