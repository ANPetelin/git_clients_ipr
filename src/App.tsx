// import { useDispatch, useSelector } from 'react-redux';

import logo from './logo.svg';
import './App.css';
// import { fetchUsers } from './redux/reducers/usersReduser';
// import { DispatchType, StateModel } from './redux/store';

export function App() {
  // const dispatch = useDispatch<DispatchType>();
  // const {
  //   loader: { loading },
  //   users: { users },
  // } = useSelector((state: StateModel) => state);

  return (
    <div className="App">
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        {/* <button onClick={() => dispatch(fetchUsers('created:>=2021-01-01'))}>TEST_FETCH</button> */}
        <p>
          Edit <code>src/App.js</code> and save to reload.
        </p>
        <a className="App-link" href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
          Learn React
        </a>
        {/* {loading ? (
          <p>...Loading</p>
        ) : (
          <ul>
            {users?.map((user) => (
              <li key={user.id}>{user.login}</li>
            ))}
          </ul>
        )} */}
      </header>
    </div>
  );
}
