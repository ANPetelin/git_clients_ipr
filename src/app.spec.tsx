import Enzyme, { mount } from 'enzyme';
import Adapter from '@cfaester/enzyme-adapter-react-18';
import { App } from './App';

Enzyme.configure({adapter: new Adapter()});
describe('My first test', () => {
    it('test', () => {
        const component = mount(<App />);
        expect(component).toMatchSnapshot();
        expect(Math.max(1, 5, 10)).toBe(10);
    })
});