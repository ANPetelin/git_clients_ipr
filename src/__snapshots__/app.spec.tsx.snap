// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My first test test 1`] = `ReactWrapper {}`;
